paf-mvp-frontend/src/lib/paf-lib.ts

```diff
@@ -178,7 +179,18 @@ export const refreshIdsAndPreferences = async ({
 
     // 1. Any Prebid 1st party cookie?
     const strIds = getCookieValue(Cookies.identifiers);
+    const lestRefresh = getCookieValue(Cookies.lastRefresh);
     const strPreferences = getCookieValue(Cookies.preferences);
+    const currentPafData = fromClientCookieValues(strIds, strPreferences);
+
+    const triggerNotification = (freshConsent: boolean) => {
+      const currentlySelectedConsent = currentPafData.preferences?.data?.use_browsing_for_personalization;
```

Comment:
NIT: move this line outside of the `triggerNotification` function.
I believe it would make it more readable.